src/server/conn.rs

```diff
@@ -1028,6 +1031,7 @@ pub(crate) mod spawn_all {
         S: HttpService<Body>,
         E: ConnStreamExec<S::Future, S::ResBody>,
         S::ResBody: 'static,
+        <S::ResBody as HttpBody>::Data: Send,
```

Comment:
This is on `NoopWatcher`. It is a `pub` item but it is not actually reachable from outside the crate. It is used in the `Future` impl for `Server<I, S, E>`, in a bound on `E`.